plugins/com.google.cloud.tools.eclipse.test.logback/.project

```diff
@@ -11,17 +11,17 @@
 			</arguments>
 		</buildCommand>
 		<buildCommand>
-			<name>org.eclipse.m2e.core.maven2Builder</name>
+			<name>org.eclipse.pde.ManifestBuilder</name>
```

Comment:
We probably shouldn't be checking these autogenerated files into the repo in the first place. They cause churn for no particular benefit. let me see if I can revert it.